### PR TITLE
Patient view

### LINKED ISSUE #260: Add Classification of SCI Injury to ASIA form
Need to add Classification of SCI Injury to ASIA Form: High Tetra (C1-C4, A,B,C), Low Tetra (C5-C8, A,B,C), Para (A,B,C), ASIA D (at Start)

Right now it is a field separate from the ASIA Widget

### LINKED ISSUE #277: UI nits that are easy to fix in one PR
distinguish title bar w/ SCI interim from nav items
sometimes we say "location" / sometimes "location name"

combine two boxes into 
"SCI hub and physician" boxes
and then ... 
assigned / preferred if different

move VA Primary Care Physician radio into box with VA and Non VA facility information

there are two blank lines for neurological level dropdown?
fix "on th job training" - typo
remove "check if" from "receiving HHA"

asia has a check box alignment issue

is inpatient should be yes/no instead of true/false
make gray boxes for future assessments not links
add link / cursor to show link as active in side bar

### LINKED ISSUE #286: Users should be associated with a location
Location can then be leveraged to auto-populate fields as appropriate.

For example, the MIO for Long Beach is almost never entering data for another facility.

### LINKED ISSUE #296: UI Update? Move Residency Type field in New Patient page?
Right now in the app, Residency Type is in the same section as Psychosocial. According to Peter's spreadsheet, Residency type should be in the same section as Caregiver so the Caregiver section should be: Caregiver and Residency. Right now in the app we have "Psychosocial and Residency".

1) Change this to Psychosocial Assessment. 
2) Move Residence Type to Caregiver Section
3) Change Cargeiver Section to Discharge Information.

### PR DESCRIPTION
**Pull request by emilyville**
fixes #296, 
fixes #277 , 
fixes #260  
Clean up / remove some old scss files from pre-emily
change title of primary scss file to "global" 
change load order of scss to be better
incorporate (some of) fatima's suggestions
side panel better
nav tweeks
made labels more consistent

make submit buttons consistent across forms
make a new class for time-span instrument (i.e START, 90 days) titles for instruments to make cleaner
remove extra boxes for instruments

### CODE REVIEW

**Comment by emilyville** [comment]
Tests are passing for me locally, confused about why they are failing here.

File: spec/features/episodes_of_care/omrs_spec.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@
     expect(page).to have_content "Patient Basic Information"
     expect(page).to have_content patient1.ssn
 
-    click_link("OMR")
+    click_link("+ OMR"))
```
Comment (awong-dev) [inline spec/features/episodes_of_care/omrs_spec.rb]: extra right paren.

**Comment by awong-dev** [comment]
@emilyville https://travis-ci.org/department-of-veterans-affairs/SCIInterim/builds/77597765#L620 shows the error. Uncertain how your tests could be passing locally...

**Comment by awong-dev** [comment]
Does this actually fix #286? If not, please update comment so we don't close the issue on merge.

**Comment by emilyville** [comment]
@awong-dev good catch. the commit was right (277) and the comment was wrong!

**Comment by emilyville** [comment]
Ok, all issues fixed aside from tests, which have a routing error - travis says only for Acute Rehab and Annual Eval, but I am _also_ get it locally for OMR.

It's saying "No route matches [PATCH] "/patients/2/acute_rehabs". I can't think of any reason my changes would have caused this.

File: app/views/layouts/patient_edit.html.slim [hunk 2]
```diff
@@ -8,56 +8,64 @@
...
-              p.pull-left
-                a 3/25/2015
-
+                td 
+                  = episode.class.name
+                  .small
+                    | completion date: 
+                    / TODO: Make sure this always exists
+                    / = episode.eval_completed
+          - if @patient.episode_of_cares.size < 1
```
Comment (awong-dev) [inline app/views/layouts/patient_edit.html.slim]: Use `@patient.episode_of_cares.empty?` instead to be more idiomatic.

File: app/views/shared/_asia_fields.html.slim [hunk 3]
```diff
@@ -35,22 +34,26 @@
                   label: false,
                   collection: Asia.collections[:neurological_motor_level_right]
       .col-md-4
+        = wrapped_form_for(@patient) do |f|
```
Comment (awong-dev) [inline app/views/shared/_asia_fields.html.slim]: This is what's causing the form submit to break.  wraped_form_for creates another <form> element within the embedding <form> and confuses all the later code.

If we have to do this, we should pass in the parent form builder as another variable other than |f|, however I'm a bit confused at the UI here.  Should the `sci_type` be per patient, or per Asia evaluation? If it's per patient, having it on the asia widget is confusing...

**Pull request merged**